modules/btcpayserver.nix

```diff
@@ -84,6 +84,11 @@ in {
         default = null;
         description = "The lightning node implementation to use.";
       };
+      rootpath = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = "The root path in the URL to access BTCPay.";
```

Comment:
```suggestion
        example = "https://mynode.org/btcpayserver/"
        description = "The root path prefix in the btcpayserver URL.";
```

Reply:
It's actually just the `/btcpayserver` addendum to the URL. 

Fixed in 93f745382912c8b37d5a2649f8ee9e085d4b1cdd